source/reference/minio-mc-admin/mc-admin-prometheus-generate.rst

```diff
@@ -90,18 +129,97 @@ Global Flags
    :end-before: end-minio-mc-globals
 
 
-Example
--------
+Examples
```

Comment:
Tried to add examples that would clarify the complexity of invoking two incompatible APIs with the same command. Suggestions welcome.